src/core/components/courses/CourseHistory.vue

```diff
@@ -76,6 +81,18 @@ export default {
     getSlotCreationDetails () {
       return get(this.courseHistory, 'slot.address.fullAddress', 'Pas d\'adresse renseignée');
     },
+    getSlotDeletionTitle () {
+      const date = this.$moment(this.courseHistory.slot.startDate).format('DD/MM');
+
+      return { pre: 'Suppression du', type: 'créneau', post: 'du', infos: `${date}` };
+    },
+    getSlotDeletionDetails () {
+      const address = get(this.courseHistory, 'slot.address.fullAddress', '');
+
+      return `Créneau initialement prévu de ${formatHoursWithMinutes(this.courseHistory.slot.startDate)}`
+        + ` à ${formatHoursWithMinutes(this.courseHistory.slot.endDate)}, `
+        + `${address ? `au ${address}` : 'Pas d\'adresse renseignée'}`; // DEMANDER à SOPHIE
```

Comment:
On n'a pas réussi à sauter de ligne ici (avec `\n`), tu sais pourquoi ?

Reply:
en combinant ces 2 articles : https://forum.vuejs.org/t/how-to-do-new-line-in-vue-js/17283/6 & https://developer.mozilla.org/fr/docs/Web/CSS/white-space on peut ajouter `white-space: pre-line` et on obtient ce qu'on veut j'ai l'impression en mettant un \r\n :)

Reply:
top !
done